react/features/base/logging/middleware.js

```diff
@@ -141,6 +141,19 @@ function _initLogging({ dispatch, getState }, loggingConfig, isTestingEnabled) {
         const _logCollector
             = new Logger.LogCollector(new JitsiMeetLogStorage(getState));
 
+        const { apiLogLevels } = getState()['features/base/config'];
+
+        if (apiLogLevels && apiLogLevels instanceof Array) {
```

Comment:
Can you please surround this with some `typeof APP !== undefined` check? Otherwise it will run on mobile and there is no APP there.